modules/administration-guide/partials/proc_configuring-gitlab-oauth2.adoc

```diff
@@ -14,7 +14,51 @@ OAuth2 for GitLab allows accepting factories from private GitLab repositories.
 
 .Procedure
 
-* Create a link:https://docs.gitlab.com/ee/integration/oauth_provider.html#authorized-applications[Authorized OAuth2 application in GitLab] using {prod-short} as the application `Name` and {identity-provider} GitLab endpoint URL as the value for `Redirect URI`. The callback URL default value is `++https://++keycloak-{prod-namespace}.__<DOMAIN>__/auth/realms/{prod-deployment}/broker/gitlab/endpoint`, where `__<DOMAIN>__` is {orch-name} cluster domain. Store the `Application ID` and `Secret` values. 
+* If the {devworkspace} engine is enabled, follow these steps:
```

Comment:
A `If` statement is a definitive sign that we should rather have 2 procedures and an assembly binding them.

* assembly_configuring-gitlab-oauth2.adoc 
* proc_configuring-gitlab-oauth2-with-the-che-server-engine.adoc 
* proc_configuring-gitlab-oauth2-with-devworkspace.adoc 

Are you feeling confident enough to do the reorganization, or do you need assistance from someone in the Docs Team?

Reply:
I will try to reorganize it myself. I'll let you know if I get stuck at something.

Reply:
I split it into two seperate procedures but wanted to keep it in the configuring authorization assembly. Let me know what you think.

Reply:
@themr0c, we're still missing the assembly that you suggested, but I looked at the build preview and it looks okay. I'm adopting proc_configuring-gitlab-oauth2-with-devworkspace.adoc for main anyway (with some editing). So, if this is okay with you, we can go ahead on this PR after my suggested changes are fixed.